### PR TITLE
Add host volume with TLS certs.

### LINKED ISSUE #120: Add files needed for TLS to Heapster container
Add the host volumes needed to send TLS requests in Heapster. This is needed by some sinks like GCM.

### PR DESCRIPTION
**Pull request by vmarmol**
Fixes #120.

### CODE REVIEW

File: deploy/heapster-controller.yaml [hunk 1]
```diff
@@ -18,6 +18,15 @@ desiredState:
               - 
                 name: "INFLUXDB_HOST"
                 value: "monitoring-influxdb"
+            volumeMounts:
+              - name: ssl-certs
+                mountPath: /etc/ssl/certs
```
Comment (vishh) [inline deploy/heapster-controller.yaml]: Noob question: Is this the default location for storing certs for most applications?

**Comment by vishh** [comment]
LGTM.

File: deploy/heapster-controller.yaml (hunk 1 above)
Comment (vmarmol) [inline deploy/heapster-controller.yaml]: From documentation I've found and references by @kelseyhightower, that is indeed the case.

Comment (vishh) [inline deploy/heapster-controller.yaml]: Ok.

**Comment by vishh** [comment]
Thanks for the PR @vmarmol. Merging.

**Pull request merged**